.github/workflows/mirror.yml

```diff
@@ -0,0 +1,30 @@
+name: Trigger Mirror
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Let's run it on all branches, so that the repo also gets mirrored during development.